.github/workflows/windows.yml

```diff
@@ -62,3 +62,9 @@ jobs:
     - name: 'test ignored'
       run: cargo test -- --ignored
       shell: cmd
+    - name: 'test rustfmt-core'
+      run: cargo test --manifest-path rustfmt-core/Cargo.toml
+      shell: cmd
+    - name: 'test rustfmt-core ignored'
+      run: cargo test --manifest-path rustfmt-core/Cargo.toml -- --ignored
+      shell: cmd
```

Comment:
Another enhacement we could consider making with GHA in the future would be to set up some additional workflows that have directory/crate specific triggers. That way if the only changes are to the emitter crate, our CI will only have to build the emitter crate